Default.sublime-commands

```diff
@@ -83,19 +70,17 @@
     {
         "caption": "LSP: Next Diagnostic",
         "command": "lsp_next_diagnostic",
-        "args": {}
     },
     {
         "caption": "LSP: Previous Diagnostic",
         "command": "lsp_previous_diagnostic",
-        "args": {}
     },
     {
-        "caption": "LSP: Workspace Symbol",
+        "caption": "LSP: Goto Symbol In Project",
```

Comment:
Instead of `LSP: Goto Symbol In Project`
I would suggest:
- `Workspace Symbols` (like before, maybe familiar to other people coming from VS Code, Emac, Vim...)
- `Project Symbols` (new, but shorter).

Reply:
I would like to avoid the word "workspace" because it is not ST-native. Synonymous words for ST are "window" or "project".

I'll change it to "Project Symbols", but it should really be a substitute for ST's native "Goto Symbol In Project"